Cubical/Foundations/Isomorphism.agda

```diff
@@ -201,6 +201,11 @@ codomainIso : ∀ {ℓ ℓ' ℓ''} {A : Type ℓ} {B : Type ℓ'} {C : Type ℓ'
            → Iso (A → B) (A → C)
 codomainIso z = codomainIsoDep λ _ → z
 
+endoIso : Iso A B → Iso (A → A) (B → B)
+endoIso is = compIso (domIso is) (codomainIso is)
+
+opIso : Iso A B → Iso (A → A → A) (B → B → B)
```

Comment:
The name is very unclear for me, maybe at least `binaryOpIso`?

Reply:
Sure, changed.